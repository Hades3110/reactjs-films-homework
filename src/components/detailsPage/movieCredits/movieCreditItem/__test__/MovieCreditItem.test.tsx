import { create } from "react-test-renderer";
import MovieCreditsItem from '../MovieCreditItem';

describe('MovieCreditsItem Component', () => {

  it('MovieCreditsItem snapshot test', () => {
    const component = create(<MovieCreditsItem character={''} gender={0} name={''} profile_path={''} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});