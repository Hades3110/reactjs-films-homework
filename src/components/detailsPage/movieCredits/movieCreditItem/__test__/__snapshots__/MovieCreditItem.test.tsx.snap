// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCreditsItem Component MovieCreditsItem snapshot test 1`] = `
<div
  className="card"
>
  <img
    alt=""
    src="test-file-stub"
  />
  <div
    className="card__info"
  >
    <div>
      Character: 
      
    </div>
    <div>
      Name:
      
    </div>
    <div>
      Sex: 
      Men
    </div>
  </div>
</div>
`;
